kubernetes/ingress/wildcamgorongosa.org.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: certmanager.k8s.io/v1alpha1
 kind: Certificate
 metadata:
   name: wildcamgorongosa-org-tls
-  labels:
-    use-azuredns-solver: "true"
 spec:
   issuerRef:
     name: letsencrypt-prod
     kind: ClusterIssuer 
   secretName: wildcamgorongosa-org-tls
   dnsNames:  
     - wildcamgorongosa.org
-    - "*.wildcamgorongosa.org"
+    - www.wildcamgorongosa.org
```

Comment:
Not blocking: Do we own these domains? is it better to leave the wildcard in for future subdomains?

Reply:
We don't own them. It would be easier to add subdomains later if we had wildcard entries, but we can only get wildcard certs with DNS validation and I want to get these moved off of CloudFront without having to wait for the project teams to update the name servers.